test/e2e/apimachinery/watch.go

```diff
@@ -43,7 +41,7 @@ const (
 var _ = SIGDescribe("Watchers", func() {
 	f := framework.NewDefaultFramework("watch")
 
-	It("should concurrently observe add, update, and delete events on pods", func() {
+	It("should observe add, update, and delete events on pods", func() {
```

Comment:
what's the reason to make it serial?

Reply:
I think it used too much code just maintaining the concurrency and the test should be valid either way. Also, it's not serial in the sense that it can't be run at the same time as other tests, so the length of time it takes to run shouldn't matter much.